### PR TITLE
docs(home): changes bg animation to 3d stars

### PR DESCRIPTION
**Pull request by bobylito**
Uses three.js. It is a custom build that will only be 57kB gzipped.
On top of that the three.js is loaded after everything else.

A small gif to give you an idea (the gradients don't display well :( )
starsystem

### CODE REVIEW

**Comment by pixelastic** [comment]
Uh, yeah ok. I have no idea what is going on inside the whole particule code, but that looks nice :D

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -42,10 +42,12 @@
     <script src="{{ "/js/search.js" | prepend: site.baseurl }}"></script>
     <script src="//use.fonticons.com/dff9b562.js"></script> <!-- replace font-awesome -->
 {%if page.url == '/index.html' %}
-    <script src="//cdn.jsdelivr.net/gsap/1.11.8/TweenMax.min.js"></script>
-    <script src="//cdn.jsdelivr.net/gsap/1.11.8/TimelineMax.min.js"></script>
-    <script src="//cdn.jsdelivr.net/scrollmagic/2.0.5/ScrollMagic.min.js"></script>
-    <script src="//cdn.jsdelivr.net/scrollmagic/2.0.5/plugins/animation.gsap.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/gsap/1.11.8/TweenMax.min.js"></script>
```
Comment (vvo) [inline docs/_layouts/default.html]: why this change?

Comment (bobylito) [inline docs/_layouts/default.html]: I must have made a mistake here while rebasing or something. :/ will fix it.

File: docs/_layouts/default.html [hunk 2]
```diff
@@ -42,10 +42,12 @@
...
+    <script src="https://cdn.jsdelivr.net/gsap/1.11.8/TimelineMax.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/scrollmagic/2.0.5/ScrollMagic.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/scrollmagic/2.0.5/plugins/animation.gsap.min.js"></script>
+    <script src="js/three.min.js"></script>
```
Comment (bobylito) [inline docs/_layouts/default.html]: and this one is wrong too :)

**Comment by Jerskouille** [comment]
:heart:

**Comment by bobylito** [comment]
Do we merge already? :D

**Comment by vvo** [comment]
nice
